conda_build/cli/main_build.py

```diff
@@ -25,21 +26,19 @@
 import conda_build.source as source
 from conda_build.utils import LoggingContext
 from conda_build.config import Config, zstd_compression_level_default, get_channel_urls
-from os.path import abspath, expanduser, expandvars
-
-on_win = (sys.platform == 'win32')
-
-logging.basicConfig(level=logging.INFO)
 
 
 def parse_args(args):
     p = get_render_parser()
-    p.description = """
-Tool for building conda packages. A conda package is a binary tarball
-containing system-level libraries, Python modules, executable programs, or
-other components. conda keeps track of dependencies between packages and
-platform specifics, making it simple to create working environments from
-different sets of packages."""
+    p.description = dals(
+        """
+        Tool for building conda packages. A conda package is a binary tarball
+        containing system-level libraries, Python modules, executable programs, or
+        other components. conda keeps track of dependencies between packages and
+        platform specifics, making it simple to create working environments from
+        different sets of packages.
+        """
+    )
```

Comment:
I don't like that dals() processes the text every single time, at least this function is not called many times.

Reply:
Agreed, but until we decide on a policy for what to do with blocks of text (e.g. should they be triple quote strings with dedent or string concatenation with explicit `\n`), I will continue using `dals` anywhere I see paragraphs.